aws_lambda_powertools/utilities/parser/models/apigw.py

```diff
@@ -1,10 +1,10 @@
-from datetime import datetime
-from typing import Any, Dict, List, Optional, Type, Union
+from __future__ import annotations
 
-from pydantic import BaseModel, model_validator
-from pydantic.networks import IPvAnyNetwork
+from datetime import datetime  # noqa: TCH003
+from typing import Any, Literal
 
-from aws_lambda_powertools.utilities.parser.types import Literal
```

Comment:
I've changed all `from aws_lambda_powertools.utilities.parser.types import Literal` to directly import from typing to avoid the indirection internally.

Reply:
Perfect!